sw.js

```diff
@@ -32,10 +32,8 @@ workbox.routing.registerRoute(
 );
 
 workbox.routing.registerRoute(
-  // Cache immutable files forever
-  /^https:\/\/[^.]+\.wikipedia\.org\//,
-  // Use the cache if it's available
-  workbox.strategies.cacheFirst({
+  /^https:\/\/[^.]+\.(m\.)?wikipedia\.org\//,
```

Comment:
update wikipedia articles caching strategy to always "load the most recent version of the article," while online, but it will load "the last-known version of the article." while offline